frontend/src/app/modules/intervention-form/form/form.component.html

```diff
@@ -76,7 +76,10 @@ <h2 class="address--header mat-h2">Adres interwencji</h2>
         </mat-form-field>
       </section>
 
-      <ng-content></ng-content>
+      <section class="attachments">
+        <h2 class="mat-h2">Załączniki</h2>
+        <eko-file-uploader [imagesOnly]="!this.inPrivateMode"></eko-file-uploader>
```

Comment:
File uploader should be decoupled from the header and container for two reasons.
1. Possible future uploader reuse in other contexts
2. Possibility of adding another component in the future within "Załączniki" (e.g file removal component, re-captcha component etc )